scripts/stablex/utilities.js

```diff
@@ -143,6 +143,36 @@ const sendLiquidityOrders = async function(
   )
 }
 
+// Given an orderbook A:B & B:C compute orderbook B:C
+const transitiveOrderbook = function(lhs, rhs) {
```

Comment:
I think this should be done in typescript

Reply:
Yes, this slipped in by accident. It was my first draft at the algorithm until I decided to do all of it in ts. Will send a cleanup PR to remove (it's unused)